Doc/whatsnew/3.7.rst

```diff
@@ -234,6 +234,9 @@ Optimizations
   expressions <re>`.  Searching some patterns can now be up to 20 times faster.
   (Contributed by Serhiy Storchaka in :issue:`30285`.)
 
+* :meth:`selectors.EpollSelector.modify`, :meth:`selectors.PollSelector.modify`
+  and :meth:`selectors.DevpollSelector.modify` may be up to 11.4% faster under
```

Comment:
This number comes from the benchmark script provided in http://bugs.python.org/issue30581. Perhaps it's better to rephrase this as "about 10% faster"?